.github/workflows/release_nightly.yaml

```diff
@@ -1,7 +1,7 @@
 name: Release nightly
 
 on:
-  create:
+  push:
```

Comment:
With this change, the nightly build is released when someone pushes tags following the syntax rule defined under `.on.push.tags`. Why do we need this change? Is it because of:
```
Nightly release should be created only when new commits are push on master branch.
```
If so, does `.on.push` help here? Because new tags are created when we cut the new release, not we merge a commit to a master.

Reply:
`Create nightly tag` GithubAction creates a tag when we push commits on master. 
Once that tag is created, Release nightly will be triggered

Reply:
@komalsukhani  Can you please confirm if I've got this right? 😄 
`Create nightly tag` action -> creates a new tag -> `Release nightly` action creates a new nightly build once the new tag is created.

>Once that tag is created, Release nightly will be triggered

So, why are we changing `on.create` to `on.push`?

Reply:
Because Create is also triggered when new branch is created. So if you raise a PR, it will run which we don't want.

Reply:
Oh, totally makes sense. Thank you @komalsukhani